test/e2e/framework/log/logger.go

```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// TODO(oomichi): Package log should be removed after switching to use core framework log.
```

Comment:
Sorry, I misunderstood the following error message
```
test/e2e/framework/log/logger.go:17:1: package comment should be of the form "Package log ..."
```
I should remove `TODO(oomichi): ` also..